packages/components-js/tests/e2e/puppeteer/specs/carousel.e2e.ts

```diff
@@ -1141,4 +1142,15 @@ describe('accessibility', () => {
 
     await expectA11yToMatchSnapshot(page, splide, { message: 'splide', interestingOnly: false });
   });
+
+  it('should change skip-link to visible if it receives keyboard focus', async () => {
+    await initCarousel({ skipLinkTarget: '#link-after' });
+    const skipLinkHost = await getSkipLinkHost();
+
+    expect(await getElementStyle(skipLinkHost, 'opacity')).toBe('0');
+
+    await page.keyboard.press('Tab');
+
```

Comment:
Maybe also check that the focus is on the skip link

Reply:
@henrijoss  Isn't this indirectly checked with the CSS selector `:focus`? Because this selector controls the opacity which then gets checked by the test.

Reply:
I guess additionally checking the active element wouldn't hurt, since focus isn't explicitly covered. Currently this test would be green by having a keydown listener and adding a class or inline style on the element. 

Besides, what about the states vrts? These wrongly show the skip link button then since selectors are too greedy/wrong?